mkchromecast/__init__.py

```diff
@@ -33,7 +33,7 @@
     whereas, installation from source needs users to go inside the cloned git
     repository and execute:
 
-        python mkchromecast.py
+        mkchromecast
```

Comment:
Add a `./`, because not all (none?) modern Unix-like operating systems assume that the current working directory is part of the `PATH`.

Reply:
The `mkchromecast` script is suppose to be in /usr/bin and when installing the debian package is placed there.